gdal/doc/source/programs/index.rst

```diff
@@ -45,6 +46,7 @@ Raster programs
    gdalcompare
    gdal_viewshed
    gdal_create
+   gdal_coord_val
```

Comment:
This is a "duplicate" of gdallocationinfo. We don't want duplicated offficial utilities. It was fine as a python sample for people wanting to see how to do that in python

Reply:
I didn't know about that one. 
Maybe I could make it to use the same API and rename it to `gdallocationinfo.py` ?

Reply:
> Maybe I could make it to use the same API and rename it to `gdallocationinfo.py` ?

could make sense

Reply:
done